rest_social_auth/serializers.py

```diff
@@ -87,3 +87,52 @@ def get_token(self, obj):
 
 class UserKnoxSerializer(KnoxSerializer, UserSerializer):
     pass
+
+
+class SimpleJWTBaseSerializer(serializers.Serializer):
+    def __init__(self, *args, **kwargs):
+        super(SimpleJWTBaseSerializer, self).__init__(*args, **kwargs)
+        if self.instance is not None:
+            self.token = self.get_token(self.instance)
+
+    @classmethod
+    def get_token(cls, user):
+        raise NotImplementedError('Must implement `get_token` method for `SimpleJWTBaseSerializer` subclasses')
+
+
+class SimpleJWTObtainPairSerializer(SimpleJWTBaseSerializer):
+    access = serializers.SerializerMethodField()
+    refresh = serializers.SerializerMethodField()
+
+    @classmethod
+    def get_token_instance(cls, user):
```

Comment:
Shouldn't this be `get_token`?

Reply:
No, this should be `get_token_instance()` since otherwise it would conflict with the `get_token()` used by the `SerializerMethodField` in the `SimpleJWTObtainSlidingSerializer`. 
It should be changed in `SimpleJWTBaseSerializer`